frontends/web/src/locales/en/app.json

```diff
@@ -942,10 +942,6 @@
       "servers": {
         "text": "This app communicates with the Shift Crypto servers to check for updates, load transactions, and send information to paired mobile apps.\nThe app also retrieves the latest exchange rates from CryptoCompare. All conversions are calculated locally which means no data about the amount of your transaction is ever transmitted.\nNote: For Ethereum and ERC20 Tokens, we use Etherscan.io APIs.",
         "title": "Which servers does this app talk to?"
-      },
-      "whyMultipleAccounts": {
```

Comment:
This entry is still used in manage-accounts.tsx: https://github.com/digitalbitbox/bitbox-wallet-app/blob/6f725bc/frontends/web/src/routes/settings/manage-accounts.tsx#L132

Reply:
Good catch. I assume this was a placeholder entry display the guide on the page? The help text does not fit the page otherwise. @thisconnect correct?

Reply:
I removed the guide entry from the manage accounts page.

Reply:
Yes I kept this as placeholder, it can be removed.